01 Operator Overloading/OperatorOverloading.Model/InvalidCurrencyException.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OperatorOverloading.Model
+{
+    [Serializable()]
```

Comment:
just adding the Serializable attribute will not make it serializable. You need to implement the proper constructor.